src/common/lru_cache.h

```diff
@@ -551,6 +559,26 @@ void SglLRUCache<K, KeyTraits>::Put(const K &key) {
     PutLocked(key);
 }
 
+template <typename K, typename KeyTraits>
+bool SglLRUCache<K, KeyTraits>::MoveBack(const K &key) {
+    ::curve::common::WriteLockGuard guard(lock_);
+    auto iter = cache_.find(key);
+    if (iter == cache_.end()) {
+        return false;
+    }
+    // delete the old value
+    RemoveElement(iter->second);
+    // put new value at tail
+    ll_.push_back(key);
+    cache_[key] = --ll_.end();
+    size_++;
+    if (cacheMetrics_ != nullptr) {
+        cacheMetrics_->UpdateAddToCacheCount();
+        cacheMetrics_->UpdateAddToCacheBytes(KeyTraits::CountBytes(key));
+    }
+    return true;
+}
+
```

Comment:
I think the objname of the current diskcache no longer belongs to lru, it is recommended to implement this module by yourself

Reply:
ok， i have add todo in the code, will edit later.